src/main/scala/me/rexim/issuestant/splittree/SplitIssueTree.scala

```diff
@@ -1,17 +1,16 @@
-package me.rexim.issuestant
+package me.rexim.issuestant.splittree
+
+import me.rexim.issuestant.github._
 
 object SplitIssueTree {
   def fromIssue(issue: Issue) = SplitIssueTree(issue.number, issue.title, issue.html_url, Nil)
 }
 
 case class SplitIssueTree(number: Int, title: String, htmlUrl: String, children: List[SplitIssueTree]) {
-  private def printLevel(level: Int): Unit = {
+  private def toStringLevel(level: Int): List[String] = {
     val padding = (1 to level).map(_ => ' ').mkString
```

Comment:
Instead of creating a collection with `level - 1` elements and `.mkString`-ing it, you could consider using `" " * level`. It's implementation uses a mutable StringBuilder and has a better performance in comparison to create a new collection.